app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -31,9 +31,13 @@ <h1 class="govuk-heading-m">@messages("checkYourAnswers.organisationDetails")</h
 
         @govukSummaryList(organisationDetails)
 
-        <h1 class="govuk-heading-m">@messages("checkYourAnswers.contactDetails")</h1>
+        @if(contactDetails.rows.nonEmpty) {
```

Comment:
Not sure I understand why the contact details summary list os now conditional as we should always have a primary contact (and hence a contact details summary)... unless I'm missing something?

Reply:
At this moment user is still not asked about contact details if I'm correct. Those questions come later on and at this point are not present. Is that correct @paddynski-opencast ?

Reply:
Reverted this change and created a specific check your answers for Other entities.